fuzzy_file_nav.sublime-settings

```diff
@@ -16,6 +16,12 @@
     // Controls whether system hidden files are shown in FuzzyFileNav.
     "show_system_hidden_files": true,
 
+    // Controls how files will be sorted in the list. The default
+    // behavior is to show fodlers first, then files, in alphabetical order.
+    // Setting this to true will mix files and folders, and they
+    // will be sorted by the last time they were accessed.
+    "sort_entries_by_last_accessed": false,
```

Comment:
I would prefer a more simple `sort_entries` that you set to either alphabetical or last-access.